open-metadata-implementation/user-interfaces/presentation-server/nodejs/src/client/components/Dino/README.md

```diff
@@ -6,66 +6,76 @@ The Dino interface provides a way to visualize Egeria platforms, servers, the se
 
 
 ### Deployment
-The Dino interface runs under the Presentation Server component of Egeria, and it uses the Dino View Service in order to retrieve the information you ask for. For details of the Presentation Server and View Server please refer to their readme information.
+The Dino interface runs under the Presentation Server component of Egeria, and it uses the Dino View Service in order to retrieve the information you ask for. For details of the Presentation Server and View Services please refer to their readme information, located here [Presentation Server](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/user-interfaces/presentation-server/README.md) and [View Services](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/view-services/README.md).
 
 
 ### Configuration
-The Dino View Service is an Integration View Service and is configured using `IntegrationViewServiceConfig`. This contains the name and status of the View Service and also contains a list of the resources that the Dino interface is permitted to access. These resources are platforms and servers that will appear in the selectors in the Dino interface. All requests to the Dino View Service REST API are based on these configured named resources. When a user selects a platform name or server name from the selector lists, the interface sends the resource name to the Dino View Service, which resolves the platform or server name to a resource endpoint to identify the URL needed to send a request to the platform or server. 
+The Dino View Service is an Integration View Service and is configured using the admin-services API to post the configuration to `open-metadata/admin-services/users/{userName}/servers/{serverName}/view-services/{vewServiceName}`. The configuration is supplied in `IntegrationViewServiceConfig`, described below or in the View Services README.md (see above). This contains the name and operational status of the View Service and also contains a list of the resource endpoints that the Dino interface is permitted to access. This list is a list of `ResourceEndpointConfig` objects, as in the example below, that represent platforms and servers.
```

Comment:
Pointer to a definition of `IntegrationViewService` (with care to distinguish between integration service).  How to set up should be in admin guide not here.

Reply:
Again - fear comment. In the latest push to this PR I have refactored the documentation so that all configuration material is in the admin guide, pointed to from Dino docs, and have separated the module README information from the online HELP information which is a brief guide for the user on how to use Dino. (Have done the same in PR for #3840 for Rex and will do a similar refactor for Tex in a separate PR).